### PR TITLE
Fixing a bug in fam_copy:

### PR DESCRIPTION
**Pull request by gautham-bhat-k**
fam_copy was not behaving asynchonously in case of CIS direct interface.
It is fixed with this commit.

### CODE REVIEW

**Comment by rajakrishi** [comment]
Can one of the admins verify this patch?

File: CMakeLists.txt [hunk 1]
```diff
@@ -132,7 +132,7 @@ include_directories(/usr/local/include)
 set( TEST_DEFAULT_REGION "Default" )
 set( TEST_OPENFAM_MODEL "memory_server" )
 set( TEST_CIS_INTERFACE_TYPE "rpc" )
-set( TEST_CIS_SERVER "127.0.0.1" )
+set( TEST_CIS_SERVER "10.93.232.37" )
```
Comment (mashoodk) [inline CMakeLists.txt:135]: Remove these changes.

**Review by mashoodk — CHANGES_REQUESTED** [review]
minor comments..

**Review by faizan-barmawer — APPROVED** [review]
Code looks fine.

**Review by mashoodk — APPROVED** [review]

**Pull request merged**